INSTALL.rst

```diff
@@ -231,7 +231,12 @@ To run only the essential services using docker, execute the following:
     $ docker-compose up -d
 
 This should bring up four docker nodes with PostgreSQL (db), Elasticsearch (es),
-RabbitMQ (mq), and Redis (cache). Keep this shell session alive.
+RabbitMQ (mq), Redis (cache) and Statping (statping). Keep this shell session alive.
+
+.. note::
+
+   For monitoring CAP locally, make sure to run the command where you run the server
+   $ export "DEV_HOST=host.docker.internal"
```

Comment:
We should also add instructions to setup services, etc, (either with `services.yml` or something else)